whois-api/src/test/java/net/ripe/db/whois/api/ElasticSearchHelper.java

```diff
@@ -113,6 +112,11 @@ private XContentBuilder getSettings() throws IOException {
                             .field("preserve_original", true)
                             .field("split_on_case_change", true)
                         .endObject()
+                        .startObject("my_word_email_delimiter_graph")
```

Comment:
By default the values of the rest of fields are
generate_word_parts -> true
catenate_words -> false
catenate_numbers -> false

I could add those fields for readability instead of skip them. The one that is fixing the problem is generate_word_parts -> true. This will generate all the combinations of the world as far as I understand.